Update title() function documentaton to show non-working example. (#668)
hashicorp/web-unified-docs: content/terraform/v1.12.x/docs/language/functions/title.mdx

# `title` Function

> `title` converts the first letter of each word in the given string to uppercase.  It does not downcase any letters in the string.

Reviewer: Hello @bschonec! Great catch and thank you for the contribution! How would you feel about rewording this a little:
```suggestion
`title` converts the first letter of each word in the given string to uppercase. The `title` function does not change any other letters in the string.
```

Thanks for adding an example too!
